Sources/Logging/Logging.swift

```diff
@@ -498,11 +498,30 @@ public struct MultiplexLogHandler: LogHandler {
     }
 }
 
+/// A wrapper to facilitate `print`-ing to stderr and stdio
+private struct FileOutputStream: TextOutputStream {
+    let file: UnsafeMutablePointer<FILE>
+
+    func write(_ string: String) {
+        string.withCString { ptr in
+            _ = fputs(ptr, file)
```

Comment:
```suggestion
            flockfile(file)
            defer {
                funlockfile(file)
            }
            _ = fputs(ptr, file)
```

one thing I only notice now (sorry) is that Swift's `print` function uses `flockfile`/`funlockfile` to make sure that the prints from different threads don't interleave. So I think we should make this function as the above suggestion.

Reply:
Thanks for going deeper on that, I saw that `Swift.print` attempts to [lock something](https://github.com/apple/swift/blob/master/stdlib/public/core/Print.swift#L225) on the output stream, but now that I went further I see that the `_lock`. I had originally assumed the the lock was standard library magic and did not pursue, bad assumption in this case.

I'll make this change 👍